packages/program/src/types.ts

```diff
@@ -137,6 +138,21 @@ export type FunctionResult<TReturn> = {
   readonly transactionResult: TransactionResult<TransactionType.Script>;
 };
 
+export type PreConfirmationFunctionResult<TReturn> = {
```

Comment:
In this case, I can see the `c` in uppercase.

Maybe I missed something in my previous comment about it.